src/js/view/controls/controlbar.js

```diff
@@ -171,35 +171,36 @@ define([
                 }, this._localization),
                 fullscreen: button('jw-icon-fullscreen', () => {
                     _api.setFullscreen();
-                }, this._localization.fullscreen)
+                }, this._localization.fullscreen),
+                spacer: div('jw-spacer'),
+                buttonContainer: div('jw-button-container')
             };
 
-            this.layout = {
-                left: [
-                    this.elements.play,
-                    this.elements.rewind,
-                    this.elements.elapsed,
-                    this.elements.durationLeft,
-                    this.elements.countdown
-                ],
-                center: [
-                    this.elements.time,
-                    this.elements.alt
-                ],
-                right: [
-                    this.elements.duration,
-                    this.elements.next,
-                    this.elements.hd,
-                    this.elements.cc,
-                    this.elements.audiotracks,
-                    this.elements.playbackrates,
-                    this.elements.mute,
-                    this.elements.cast,
-                    this.elements.volume,
-                    this.elements.volumetooltip,
-                    this.elements.fullscreen
-                ]
-            };
+            this.buttonLayout = [
+                this.elements.play,
+                this.elements.alt,
+                this.elements.rewind,
+                this.elements.elapsed,
+                this.elements.countdown,
+                this.elements.duration,
+                this.elements.spacer,
+                this.elements.next,
+                this.elements.hd,
+                this.elements.cc,
+                this.elements.audiotracks,
+                this.elements.playbackrates,
+                this.elements.mute,
+                this.elements.cast,
+                this.elements.volume,
+                this.elements.volumetooltip,
+                this.elements.fullscreen
+            ];
+
+
+            this.layout = [
+                this.elements.time,
+                this.elements.buttonContainer
+            ];
```

Comment:
`this.elements` should be assigned to a variable to improve minification. (Search "this.elements" in http://ssl.p.jwpcdn.com/player/v/7.11.2/jwplayer.controls.js)